taxonomies/categories.txt

```diff
@@ -4563,19 +4563,27 @@ en:Fruit and milk beverages, Fruit and milk juices, milk and fruit juices, Fruit
 de:Frucht- und Milchsäfte, Milch und Fruchtsäfte
 es:Bebidas de fruta y leche
 fi:Mehu-maitojuomat
-fr:Boissons au lait et au jus de fruits, jus de fruits au lait, jus de fruits lactés, Boissons au jus de fruit et au lait
+fr:Boissons au fruit et au lait, Boissons au lait et au jus de fruits, jus de fruits au lait, jus de fruits lactés, Boissons au jus de fruit et au lait
```

Comment:
Maybe add "Boissons aux fruits et au lait" with fruits in plural as the main name? It sounds better than "Boisson au fruit" which kind of implies that there is only one fruit.

Reply:
I changed it